tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -139,17 +138,18 @@ bool CanInitialize()
 
             Task StartAsync()
             {
-                LifetimeManager.Instance.AddShutdownTask(OnShutdown);
+                AddShutdownTask();
```

Comment:
Why make this change that adds 4 tasks to the shutdown manager instead of a single task? 🤔

Reply:
Each shutdown task surrounded with `try..catch` block, and as discussed with @OmerRaviv we decided to be extra safe here, just to be sure, that every task is called.